### ISSUE TITLE
Lutris - Debian - no pipewire: Gamepad question

### ISSUE THREAD

**Issue by Juppstein** [theNizo/linux_rocksmith#18]
* Which distro do you use? UBUNTU 22.04 LTS
* Which Proton or wine version do you use? GE 3-70
* Do you use pipewire? NO

* What appears to be the problem? (Describe it as best as you can)

Just a question really. Does a gamepad work for you with the Lutris - Debian - non-pipewire solution?

https://github.com/theNizo/linux_rocksmith/blob/main/guides/setup/deb-non-pipewire-old.md

**Comment by theNizo** [comment]
Sorry, I forgot about this issue. I will adress this today in the afternoon.

**Comment by theNizo** [comment]
> I will adress this today in the afternoon.

Nevermind, my game doesn't start.

**Comment by theNizo** [comment]
Alright, got it running.

Gamepad support should not depend on the sound server. The potential issue could be your distro, but I can't remember having issues on Ubuntu with that.

To answer the question directly, my XBox 360 controller works with the game, but I don't have any debian-based distro installed right now.

**Comment by Juppstein** [comment]
Right, I'm aware that the sound engine does not influence input devices. My question was just if you've made the same observation when you tested on your end.
Anyway, in the end I had to use AntimicroX with a dedicated key-to-button mapping in order to get (Bluetooth) gamepad support with Lutris under Ubuntu 22.04 up and running . I tested with an 8bitdo 30n Pro (xpad mode) and an Xbox Elite Series 2 (with xpadneo), both with the same symptoms.

14.09.2022 21:43:53 Nizo ***@***.***>:
